pep/app/pods/read/document/controller.ts

```diff
@@ -354,6 +356,29 @@ export default class ReadDocument extends Controller {
             }
         ]);
     }
+
+    /**
+     * Clear the old search and then go to the search page using the new search terms
+     *
+     * @param {string} searchTerms
+     * @memberof ReadDocument
+     */
+    @action
+    viewSearch(searchTerms: string) {
+        // TODO improve this typing
+        clearSearch(this as any, this.configuration, this.currentUser);
+        this.router.transitionTo('search', {
+            queryParams: {
```

Comment:
you could use the configuration service's `defaultSearchParams` getter as the basis for this to dry it up a little:
```ts
queryParams: { ...this.configuration.defaultSearchParams, searchTerms }
```